src/main/scala/cromwell/binding/expression/DeclarationEvaluator.scala

```diff
@@ -0,0 +1,14 @@
+package cromwell.binding.expression
+
+import cromwell.binding.Declaration
+import cromwell.parser.WdlParser.AstNode
+
+import scala.language.postfixOps
+import scala.util.Try
+
+case class DeclarationEvaluator(override val lookup: String => Seq[Declaration], override val functions: WdlFunctions[Seq[Declaration]]) extends Evaluator {
+  override type T = Seq[Declaration]
+
+  override def evaluate(ast: AstNode): Try[Seq[Declaration]] = ???
+}
+
```

Comment:
This Evaluator wasn' t mentioned in the notes for this PR and isn't used, which given its implementation I think is a good thing.  :smile: